compiler/tests/org/jetbrains/kotlin/repl/AbstractReplInterpreterTest.kt

```diff
@@ -45,6 +47,13 @@ public abstract class AbstractReplInterpreterTest : UsefulTestCase() {
 
     private data class OneLine(val code: String, val expected: String)
 
+    override fun setUp() {
+        super.setUp()
+        // set to false if not present or set to true or unknown (empty considered true)
+        if (System.getProperty(KOTLIN_COMPILER_ENVIRONMENT_KEEPALIVE_PROPERTY)?.let { it.toBooleanLenient() ?: true } ?: true )
```

Comment:
Also, I suggest to store previous value of property and restore it in `tearDown`.